changelog.txt

```diff
@@ -1,3 +1,38 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Add: Make NewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
+Add: Display refresh rate game option (#8813)
+Add: [Video] Move GameLoop into its own thread (#8741)
+Add: Option to (dis-)allow accelerated video drivers (#8819)
+Add: Filter for "engines only" in build train window (#8733)
+Add: Malaysia Ringgit as Currency (#8783)
+Change: Recolour graph windows to brown (#8700)
+Change: Don't show global goals in company goal windows (#8709)
+Change: Clarify what effect town interactions have (#8744)
+Change: De-limit framerate window's framerate (#8772)
+Fix #8851: Don't allow infinite "exec" depth in script, but limit to 10 deep (#8852)
+Fix #8647: Order of drawing tram catenaries wasn't correct (#8843)
+Fix #8833: Don't reload NewGRFs when we are shutting down (#8830)
+Fix #8711: Having gui_zoom lower than zoom_min causes crashes (#8835)
+Fix #8810: "aircraft out of fuel" news shows the wrong place (#8832)
+Fix: Scale PIP-padding the same as regular padding (#8829)
+Fix #8817: Keep NewGRF order for object class sorting (#8818)
+Fix #8784: Using alt+enter didn't update the fullscreen toggle visibly (#8820)
+Fix #8808: [OSX, OpenGL] Crash on switching blitters due to double-mapping the video buffer (#8822)
+Fix: Broken company colours for 40bpp-blitter (#8821)
```

Comment:
This is actually a fix for `Fix #8774: Black screenshots when using 40bpp-blitter (#8791)`, so doesn't really need to be in  the changelog.